### PR TITLE
Add in desired capacity to server group operations

### PR DESCRIPTION
**Pull request by drmaas**
Added desiredSize attribute to payload. 

Create server group example:
```
curl -X POST -H "Content-Type: application/json" -d '[{"createServerGroup":{"stack":"teststack","application":"myapp","serverGroupParameters":{"instanceType":"m1.medium","image":"4e0d0b4b-8089-4703-af99-b6a0c90fbbc7","maxSize":5,"minSize":3, "desiredSize":4, "subnetId":"77bb3aeb-c1e2-4ce5-8d8f-b8e9128af651","poolId":"87077f97-83e7-4ea1-9ca9-40dc691846db","securityGroups":["sg-heat-test-1"]},"region":"REGION1","disableRollback":false,"timeoutMins":5,"account":"test"}}]' localhost:7002/openstack/ops
```

Resize example:
```
curl -X POST -H "Content-Type: application/json" -d '[ { "resizeServerGroup": { "serverGroupName": "myapp-teststack-v000", "capacity": { "min": 1, "desired": 2, "max": 3 }, "account": "test", "region": "REGION1" }} ]' localhost:7002/openstack/ops
```

Clone example:
```
curl -X POST -H "Content-Type: application/json" -d '[{"cloneServerGroup": {"source": {"serverGroupName": "myapp-teststack-v000", "region": "RegionOne"}, "serverGroupParameters":{"instanceType":"m1.medium","image":"4e0d0b4b-8089-4703-af99-b6a0c90fbbc7","maxSize":5,"minSize":3, "desiredSize":4, "subnetId":"77bb3aeb-c1e2-4ce5-8d8f-b8e9128af651","poolId":"87077f97-83e7-4ea1-9ca9-40dc691846db","securityGroups":["sg-heat-test-1"]}, "region": "RegionTwo", "account": "test"}}]' localhost:7002/openstack/ops
```

### CODE REVIEW

**Comment by drmaas** [comment]
@spinnaker/reviewers @derekolk @varikin fun stuff ^^^

File: clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/description/servergroup/ResizeOpenstackAtomicOperationDescription.groovy [hunk 1]
```diff
@@ -28,6 +28,7 @@ class ResizeOpenstackAtomicOperationDescription extends OpenstackAtomicOperation
   static class Capacity {
     int min
     int max
+    int desired
```
Comment (derekolk) [inline clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/description/servergroup/ResizeOpenstackAtomicOperationDescription.groovy]: Needs to be added when caching.

Comment (drmaas) [inline clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/description/servergroup/ResizeOpenstackAtomicOperationDescription.groovy]: Done

**Comment by lwander** [comment]
LGTM

**Pull request merged**